src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -96,8 +96,10 @@ private void testEnrollAction() throws Exception {
                 .withUserId(testData.instructors.get("CCEnrollUiT.teammates.test").googleId)
                 .withCourseId(courseId);
         InstructorCoursesDetailsPage detailsPage =
-                loginAdminToPage(coursesPageUrl, InstructorCoursesDetailsPage.class);
-        assertEquals(6, detailsPage.getStudentCountForCourse("CCEnrollUiT.CS2104"));
+
+                loginAdminToPage(browser, coursesPageUrl, InstructorCoursesDetailsPage.class);
+        assertEquals(6, detailsPage.getStudentCountForCourse());
```

Comment:
but in my repo as well as in master repo there only 2 parameters. There is no parameter such as `browser`. May be without your attention it happened.